### ISSUE TITLE
A11y_Visual Studio Code Services_Docs Java_Run and Debug_Audio Description (Pre-recorded): Audio Description is not provided for the Pre-recorded videos on the page.

### ISSUE THREAD

**Issue by manaswita-chougule** [microsoft/vscode-docs#6303]
### GitHub Tags
#WCAG1.2.5; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #Chrome; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_Apr2023; #Audio Description (Pre-recorded);

### Environment Details
Application Name: Visual Studio Code Services
URL: [https://code.visualstudio.com/](https://code.visualstudio.com/)
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Chrome Version 113.0.5672.93 (Official Build) (64-bit)

### Repro Steps

1. Hit the application URL: https://code.visualstudio.com/
2. Tab till "Docs" link and press enter.
3. Tab till "JAVA" link in the left navigation and press enter.
4. Tab till "Run and Debug" link present under the java and press enter.
5. Tab till Videos present on the page.
6. Observe that audio Description is not provided for the Pre-recorded videos on the page.

### Actual Result
Audio Description is not provided for the Pre-recorded videos on the page.
### Note
Similar issue is observed throughout the application for the pre-recorded videos.

### Expected Result
Audio Description should be provided for the Pre-recorded videos on the page.

### User Impact
Visually challenged users will not able to get the information displayed in the video, so the audio description should be provided to ensure these users will get the same information.

### Attachment
![Audio Description is not provided for the Pre-recorded videos on the page](https://github.com/microsoft/vscode-docs/assets/91586101/40cf315c-da92-40e4-b283-df33f143a755)

**Comment by rzhao271** [comment]
The issue here is that a lot of videos under the Java tutorials don't have alt text.
@gregvanl could we link in the Java content creators/maintainers? They might be able to add alt text to the videos using strategies from https://stackoverflow.com/a/63614226.

**Comment by manaswita-chougule** [comment]
#TCSReactivated; #Regressed:05-29-23;

@rzhao271 @gregvanl The alt text provided for the video will not completely explain the content shown in the video, the audio description should be provided for the video so that the visually challenged users will able to understand all the information being conveyed through the video. Thanks!

**Comment by gregvanl** [comment]
Hi @manaswita-chougule These are short videos meant as an addition to the surrounding text and do not describe new features or workflows independent of the text. 
The alt-text is deliberately brief so as not to unnecessarily repeat the surrounding text, which is an alt text best practice.

**Comment by manaswita-chougule** [comment]
#Closed; #Regressed:06-30-23;
As per discussion with SME, closing this bug and logged usability issue - https://github.com/microsoft/vscode-docs/issues/6437#top
